e2e/cypress/integration/enterprise/guest_accounts/guest_identification_ui_spec.js

```diff
@@ -180,7 +180,7 @@ describe('MM-18045 Verify Guest User Identification in different screens', () =>
         cy.get('#moreDmModal .close').click();
     });
 
-    it('Verify Guest Badge in DM header and GM header', () => {
+    it('MM-T1379 Verify Guest Badge in DM header and GM header', () => {
```

Comment:
Looking at Zephyr for 1379, it's trying to verify "Don't show guest badge next to GM in LHS". A bit confused by it, esp. since the test script only has that heading and no specific test steps.

Reply:
Reverted for now.